docs/content/components/forms.mdx

```diff
@@ -17,137 +17,117 @@ import Playground from '../../utils/Playground';
+When you need short textual information from the user such as: email address,
+first name, phone number, street name, etc.
 
-PLACEHOLDER
-- A input field must always have a placeholder, which is a short example of the information required that helps the user to understand what kind of information is needed. (For instance an Address Field could have as placeholder: Grunerstrasse 13)
-- The placeholder must be succint (up to 3-4 words)
-- The placeholder should also be contextual to the language and market
-
-| ✓ The input field must always be accompanied by a label                                                                                                    |
-| ✓ Normally, a row should contain only one text input unless they're up to two contextual information that make sense togeter (such as first and last name) |
-
-| Dont's                                                                          |
-|-------------------------------------------------------------------------------|
-| ✕ Don't use input fields for requiring longer textual information from the user |
-| ✕ Don't communicate that a input field is required                              |
-| ✕ Don't mark required fields with asterisks                                     |
-
-## Placeholder
-### A input field must always have a placeholder, which is a short example of the information required that helps the user to understand what kind of information is needed. (For instance an Address Field could have as placeholder: Grunerstrasse 13)  
-
-| Do's                                                                                                                                                     |
-|----------------------------------------------------------------------------------------------------------------------------------------------------------|
-| ✓ The placeholder must be succint (up to 3-4 words)                                                                                                 |
-| ✓ The placeholder should also be contextual to the language and market |
-
-| Dont's                                                                          |
-|-------------------------------------------------------------------------------|
-| ✕ The placeholder should not be the exactly same information as the label |
-| ✕ Don't communicate that a field is optional in the placeholder, you should do it in the label                              |
+### Usage guidelines
 
-### Props
-> Can't render prop tables for this component because it's exported with `withTheme`.
+- **Do** always accompany an input field with a visible label.
+- **Do** use one input per row, unless two fields are closely interrelated
+  (e.g. first and last name).
+- **Do not** use input fields for requiring longer textual information from the
+  user.
+- **Do not** communicate that an input field is required.
```

Comment:
I assume that optional fields should be used instead, right? I think it would make sense to add a note about this here.

Reply:
This was just a way to express that there's no need to communicate (either or the placeholder or the label) that an input field is required. It reflects the principle of utility for the user that everything on a form is required unless explicitly said so (when it's the case of option input fields we state on the variant to communicate it on the label). Makes sense?

Reply:
One improvement in that context is to definetly give this context on that "Do not"